lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSlide.js

```diff
@@ -1,4 +1,5 @@
-import CampaignController from '../campaigns/Controller';
+import CampaignController from '../campaign/Controller';
```

Comment:
ha! did it not work before when it was plural? `import CampaignController from '../campaigns/Controller';` 😱